launchpod/angular-launchpod/backend/src/test/java/com/google/launchpod/FormHandlerServletTest.java

```diff
@@ -81,14 +81,13 @@
   @Rule // JUnit 4 uses Rules for testing specific messages
   public ExpectedException thrown = ExpectedException.none();
```

Comment:
it seems like the method .none() is deprecated. Might seem like  Assert.assertThrows might do the trick

Reply:
I think Assert.assertThrows is part of JUnit 5, I'm using JUnit 4 just according to walkthrough guidelines for unit tests since a lot of Google still uses JUnit 4